From 76373372c09b1bfdc07f6bd6e67d3913711c877e Mon Sep 17 00:00:00 2001
From: sukanka <su975853527@gmail.com>
Date: Sun, 18 May 2025 10:49:36 +0800
Subject: [PATCH 1/5] fix package version

no need to pin versions of these packages

2	2	CMakeLists.txt

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 40a9b9ee1..3b07e19ca 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -300,7 +300,7 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
 # Enforce the search mode of non-required packages for better and shorter failure messages
 find_package(enet 1.3 MODULE)
 find_package(fmt 10 REQUIRED)
-find_package(LLVM 17.0.2 MODULE COMPONENTS Demangle)
+find_package(LLVM MODULE COMPONENTS Demangle)
 find_package(lz4 REQUIRED)
 find_package(nlohmann_json 3.8 REQUIRED)
 find_package(Opus 1.3 MODULE)
@@ -345,7 +345,7 @@ endif()
 
 if (ENABLE_WEB_SERVICE)
     find_package(cpp-jwt 1.4 CONFIG)
-    find_package(httplib 0.12 MODULE COMPONENTS OpenSSL)
+    find_package(httplib MODULE COMPONENTS OpenSSL)
 endif()
 
 if (YUZU_TESTS)
-- 
2.49.0

